src/views/playground/PlaygroundLayout.tsx

```diff
@@ -19,7 +26,13 @@ export const PlaygroundView = () => {
               <Link to="./uploading-files-data">Uploading Files Data</Link>
               <Link to="./member-active-channel">Active user/member/channel</Link>
             </LinksContainer>
-            <PlaygroundRouter />
+            <Routes>
+              <Route path="/validation-form" element={<PlaygroundValidationForm />} />
+              <Route path="/drafts" element={<PlaygroundDrafts />} />
+              <Route path="/video-metadata" element={<VideoMetaData />} />
+              <Route path="/uploading-files-data" element={<PlaygroundUploadingFilesData />} />
+              <Route path="/member-active-channel" element={<PlaygroundMemberChannel />} />
+            </Routes>
```

Comment:
By analogy to StudioLayout we could replace this with `playgroundRoutes`

Reply:
:+1: cleaned this up